modules/rest-api/pages/rest-xdcr-identify-target-cluster.adoc

```diff
@@ -0,0 +1,180 @@
+= Identifying the Target in Replication
+:description: pass:q[]
+
+[abstract]
+{description}
+
+== HTTP Method and URI
+
+----
+GET /xdcr/sourceClusters
+----
+
+== Description
+
+When on a current cluster, XDCR lets you:
+
+* Retrieve the list of remote clusters that replicate to the current cluster.
+If XDCR finds a list of replicating source clusters, it makes the current cluster the target cluster.
+
+* Inspect the incoming XDCR replication streams on current cluster from each remote cluster.
+It makes the current cluster the target cluster and remote cluster the source cluster.
```

Comment:
Remove line 21 - "It makes the current cluster the target cluster and remote cluster the source cluster."  
This line is not needed, and it's confusing.